lib/main.dart

```diff
@@ -33,9 +33,7 @@ void main() {
   };
 
   final config = FLog.getDefaultConfigurations();
-  // TODO: Rethink whether we need to show Rust logs from Flutter natively or
-  // via Flutter
-  // config.activeLogLevel = foundation.kReleaseMode ? LogLevel.INFO : LogLevel.DEBUG;
+  config.activeLogLevel = foundation.kReleaseMode ? LogLevel.INFO : LogLevel.DEBUG;
```

Comment:
❓ Commit message says `trace` but I see `info` and `debug` here. Is that intentional?

Reply:
reworded the commit message, thanks for spotting!